src/System.Security.Cryptography.Pkcs/tests/Pkcs12/Pkcs12InfoTests.cs

```diff
@@ -14,11 +14,32 @@ namespace System.Security.Cryptography.Pkcs.Tests.Pkcs12
 {
     public static class Pkcs12InfoTests
     {
-        [Fact]
-        public static void ReadEmptyPfx()
+        private static ReadOnlyMemory<byte> PadContents(ReadOnlyMemory<byte> contents, int trailingByteCount)
```

Comment:
Nit: PadRight to match the naming on string, and return byte[] ?

Reply:
I looked into this:  Once the name matches I kind of want to make the int parameter change purposes to match (string's is total length, not how much to pad by).

As for the return type, the input values are ROMbyte, not byte[]; so changing to be byte[] as a return means a) making a copy at pad-0, b) changing the current properties to be byte[] and do the "it's internal, I promise to myself I won't mutate it".

Since the changes are only marginally better, and this has already passed CI, I think I'll just leave it (and then probably be annoyed with myself at the next time I change this area)

Reply:
> As for the return type, the input values are ROMbyte, not byte[]; so changing to be byte[] as a return means a) making a copy at pad-0, b) changing the current properties to be byte[] and do the "it's internal, I promise to myself I won't mutate it".

I don't understand this.  It would literally not affect the implementation code at all, since you're already allocating and returning an array and currently just relying on an implicit cast from array to ROM that would no longer happen.

Reply:
> It would literally not affect the implementation code at all

Oh, you're talking about the trailingByteCount == 0 case? Meh. Ok.